src/main/java/pl/javastart/task/Televisor.java

```diff
@@ -0,0 +1,19 @@
+package pl.javastart.task;
+
+public class Televisor {
+
+    private boolean isTurnedOn;
+
+    public void turnOn() {
+        isTurnedOn = true;
+    }
+
+    public void turnOff() {
+        isTurnedOn = false;
+    }
+
+    public boolean showStatus() {
+        return isTurnedOn;
+    }
```

Comment:
show sugeruje, że metoda coś wyświetla. Jeżeli metoda coś zwraca, to w nazwie stosujemy najczęściej czasownik "get"